### PR TITLE
Refacto front

### PR DESCRIPTION
**Pull request by emmaafrn**
Holà this is my code with the good things and the less good things (like my modals windows...)

### CODE REVIEW

**Review by DoomDuck — COMMENTED** [review]
Modal doesn't have a clear way of interacting with others.
It emits an event and also exposes isOpenModal

File: src/front/Friend.svelte [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
   export let image = "img/canard.jpeg";
   export let friendName = "Joey";
+
+  function openModal() {
+    isOpenModal = true;
+  }
+
+  function closeModal() {
+    isOpenModal = false;
+  }
```
[the remark this answers is not in the document]
Comment (emmaafrn) [reply src/front/Friend.svelte:18]: NOOOO you suck !
```svelte
<Modal {isOpenModal} {friendName} {image} on:closeModal={() => isOpenModal = false} />
```
Should be used instead

**Review by DoomDuck — CHANGES_REQUESTED** [review]
You have a warning when you compile the project

**Review by DoomDuck — CHANGES_REQUESTED** [review]
Your alt= attributes are quite strange, those are the fields read for accessibility reasons

File: src/front/Profile.svelte [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<img class="roundedImageProfile" src={image} alt="profilepic" />
+<br />
+<h1>{friendName}</h1>
+
+<style>
+  h1 {
+    color: #ff00b8;
+    font-size: 1em;
+    text-align: center;
+  }
```
[the remark this answers is not in the document]
Comment (emmaafrn) [reply src/front/Profile.svelte:15]: <h1> changed to <h3> but I don't understand why my alt attribute is strange,
I thing it's a pretty good description of what my  element is.

  <div id="Profile">
    
    profilepic
    <h3>{friendName}</h3>
    <p> iD: {iD}</p>
  
  </div>

**Review by DoomDuck — APPROVED** [review]
It's a lot better now I like it :grin:

**Review by DoomDuck — APPROVED** [review]

**Pull request merged**